### ISSUE TITLE
Support with Meteor 1.2

### ISSUE THREAD

**Issue by thearabbit** [hitchcott/app-dump#22]
Support with Meteor 1.2

**Comment by HelloLudger** [comment]
when I try to add it to my 1.2 (Windows) installation I get

```
C:\Projekte\spielm-it>meteor add hitchcott:app-dump
 => Errors while adding packages:

While checking for hitchcott:app-dump@0.1.3:
error: No compatible binary build found for this package. Contact the
package author and ask them to publish it for your platform.
```

**Comment by IstoraMandiri** [comment]
Thanks @msand , released `0.4.2`
